packages/block-editor/src/components/inspector-controls/slot.js

```diff
@@ -37,17 +34,12 @@ export default function InspectorControlsSlot( {
 	const slotFill = groups[ group ];
 	const fills = useSlotFills( slotFill?.name );
 
-	const motionContextValue = useContext( MotionContext );
-
 	const computedFillProps = useMemo(
 		() => ( {
 			...fillProps,
-			forwardedContext: [
-				...( fillProps?.forwardedContext ?? [] ),
-				[ MotionContext.Provider, { value: motionContextValue } ],
-			],
+			forwardedContext: [ ...( fillProps?.forwardedContext ?? [] ) ],
 		} ),
-		[ motionContextValue, fillProps ]
+		[ fillProps ]
```

Comment:
You can remove this entire `computedFillProps` calculation. It used to add `MotionContext` to the `forwardedContext` but now it does nothing. Just use the `fillProps` as they are.

Reply:
I thought about this, and I was wondering if it could be useful to keep the current structure of the code around in case more contexts needed to be forwarded. Do you think there's value in keeping it around?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/67522/commits/07ddcb67d41395eff08611669b6416cf7408d8ff

Reply:
You can add a comment: if you want to add a forwarded context provider, modify `fillProps` with code like this. But the actual code shouldn't be there until it does something.